src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/CacheCollectorPass.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of php-cache\cache-bundle package.
+ *
+ * (c) 2015-2015 Aaron Scherer <aequasi@gmail.com>, Tobias Nyholm <tobias.nyholm@gmail.com>
```

Comment:
Sorry, Copy/paste issue.. An my IDE does not show file headers. I will fix these